templates/download/server/power.html

```diff
@@ -1,106 +1,126 @@
 {% extends "download/_base_download.html" %}
```

Comment:
The only change to this file was the addition of the closing tag on line 60, everything else is linting